src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1613,6 +1626,15 @@ public Integer getRetryWaitTime() {
         return retryWaitTime;
     }
 
+    public boolean getTcpNoDelay() {
+        return tcpNoDelay != null ? tcpNoDelay : true;
```

Comment:
it is not possible to have a null value because of the readResolve method, but it is better to protect it